megcruzen/browserify_productOrdering
# Reviews now dynamically added to product section via fetch

# Description
Reviews are now dynamically added to product section using information in database. 

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout set-up-review-data`
3. Run server/grunt
4. Verify that two product reviews show up inside product box.

# Checklist:
